Nginx-Certbot.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Configure Nginx and TLS Certifications on Your Server
+layout: topic
+categories: guide
+---
+
+Now that you have a domain and a web server, you need to configure 
+the server to serve content to your domain.
+
+The two most common web server software suites are 
+[`nginx`](https://nginx.org/) and [`apache`](https://www.apache.org/). 
+This wiki page will cover web server configuration with `nginx` 
+because it is easier to configure for first-time users. The wiki page 
+assumes that the machine used to serve content is a Digital Ocean 
+droplet, but the specific steps won't differ widely between services.
+
+## Step 1: Install `nginx`
+
+Installation is relatively easy.
+- ssh into your droplet.
+- To confirm that the software is not currently installed, run the 
+  following command:<br>
+  ```
+  $ which nginx
+  $
+  ```
+  <br>
+  If you see no executable path before the prompt reappears, then you 
+  will need to install `nginx`.
+- Install `nginx` with the following command:<br>
+  ```
+  $ sudo apt-get install nginx
+  ```
+  <br>
+  When prompted with `Do you want to continue? [Y/n]`, type yes and 
+  press return.
+- To confirm that `nginx` was installed, rerun `which nginx`.
+  You should now see a path to the executable in the console.
+
+## Step 2: Get Your Website Code onto the Server
+
+Assuming that your code is in a `git` repository, clone the repository 
+onto your machine.
+
+In our example, we will be serving 
+[this GitHub repository](https://github.com/UWB-ACM/website-example-2) 
+at [phry.me](http://phry.me).
+
+To clone the repository, run the following commands and replace the 
+GitHub URL with the URL to your own repository.
+
+```
+$ cd /var/www
+$ git clone https://www.github.com/UWB-ACM/website-example-2.git
+$ cd website-example-2
+$ pwd
+/var/www/website-example-2
+```
+
+Note: the output for the `pwd` command will be important for the 
+next step.
+
+## Step 3: Set Up Configuration for `nginx`
+
+Now that you have the source code for your website on your machine, 
+you need to point `nginx` to the directory of the source code and 
+ensure that the content gets served on your domain.
+
+The configuration file your your site will be located under `/etc/nginx/sites-available`. On a fresh installation of `nginx`, a 
+site configuration file called `default` is provided as an example.
+
+Run the following commands and replace `phry.me` with the domain name 
+you chose. _Note: this isn't a strict requirement, but is a common 
+naming convention. It is recommended to use naming conventions._
+
+```
```

Comment:
```suggestion
```console
```

Reply:
Fixed in 7c1a9d051f9c55fa86e76013b274a40a644cea45